app/controllers/snippets_controller.rb

```diff
@@ -0,0 +1,28 @@
+class SnippetsController < ApplicationController
+  def create
+    Snippets::Create.(params: snippet_params.to_h) do |m|
+      m.failure :validate do |result|
+        redirect_to root_path
```

Comment:
Shall we add some notice so that to explain what is wrong?

I aslo suggest adding HTML required attribute to the textarea:

https://www.w3schools.com/tags/att_textarea_required.asp

Reply:
Validation errors are available in this block in `result` variable. We can notice user. This requires some kind of UI first

Reply:
Will do just for now